### DISCUSSION TITLE
svc  not found on windows 11

### DISCUSSION THREAD

**Discussion by adamgyongyosi** [voicepaw/so-vits-svc-fork#1189] [General]
![image](https://github.com/voicepaw/so-vits-svc-fork/assets/5511526/5f93fd6f-9ac7-48f2-8d3a-7294ffa5fa99)

![image](https://github.com/voicepaw/so-vits-svc-fork/assets/5511526/aa0bf825-46c8-4a3c-b6ef-7ec36185552e)

![image](https://github.com/voicepaw/so-vits-svc-fork/assets/5511526/c34f158a-8dad-4d57-b2f2-8f56c34c39a6)

**Comment by 34j** [comment]
Updating to 4.2 should fix this
